types/index.d.ts

```diff
@@ -1,87 +1,89 @@
 // Type definitions for Chalk
 // Definitions by: Thomas Sauer <https://github.com/t-sauer>
+export const enum Level {
```

Comment:
Using `const` here will cause replacements to occur during compilation -- Since this value doesn't exist at runtime